HW_6/model_base.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel, Field
+from datetime import datetime, date
+
+class User(BaseModel):
+    id: int = Field(default=None)
+    name: str = Field(min_length=2, max_length=50)
+    surname: str = Field(min_length=2, max_length=50)
+    email: str = Field(min_length=4, max_length=50)
+    password: str = Field(min_length=5, max_length=120)
+
+class UserIn(BaseModel):
+    name: str = Field(..., min_length=2, max_length=50)
+    surname: str = Field(..., min_length=2, max_length=50)
+    email: str = Field(..., min_length=4, max_length=50)
+    password: str = Field(..., min_length=5, max_length=120)
```

Comment:
Хотя, очевидно, что модели практически идентичны, значит одна может наследоваться от другой
```suggestion
class UserIn(BaseModel):
    name: str = Field(..., min_length=2, max_length=50)
    surname: str = Field(..., min_length=2, max_length=50)
    email: str = Field(..., min_length=4, max_length=50)
    password: str = Field(..., min_length=5, max_length=120)

class User(UserIn):
    id: int = Field(default=None)
```